src/test/python_tests/test_code_actions.py

```diff
@@ -21,7 +21,23 @@
     [
         (
             "C0301:line-too-long",
-            "FRUIT = ['apricot', 'blackcurrant', 'cantaloupe', 'dragon fruit', 'elderberry', 'fig', 'grapefruit']",  # # pylint: disable=line-too-long
+            # # pylint: disable=line-too-long
+            "FRUIT = ['apricot', 'blackcurrant', 'cantaloupe', 'dragon fruit', 'elderberry', 'fig', 'grapefruit']",
+            {
+                "title": f"{LINTER}: Run document formatting",
+                "command": "editor.action.formatDocument",
+                "arguments": None,
+            },
+        )
+    ],
+)
+@pytest.mark.parametrize(
+    ("code", "contents", "command"),
+    [
+        (
+            "C0305:trailing-newlines",
+            # # pylint: disable=trailing-newlines
+            "VEGGIE = ['carrot\n', 'radish\n', 'cucumber\n', 'potato\n', '\n']",
```

Comment:
You don't need to add a new parametrize decorator. you can just add to existing list.
```suggestion
            # # pylint: disable=line-too-long
            "FRUIT = ['apricot', 'blackcurrant', 'cantaloupe', 'dragon fruit', 'elderberry', 'fig', 'grapefruit']",
            {
                "title": f"{LINTER}: Run document formatting",
                "command": "editor.action.formatDocument",
                "arguments": None,
            },
        ),
        (
            "C0305:trailing-newlines",
            # # pylint: disable=trailing-newlines
            "VEGGIE = ['carrot\n', 'radish\n', 'cucumber\n', 'potato\n', '\n']",
```